### PR TITLE
PP-7859 Metadata Transactions filter - add UI elements

### PR DESCRIPTION
**Pull request by iqbalgds**
- Update controller - to pass down 'isExperimentalFeaturesEnabled' to view.
- Add 'Advance filters' toggle.
  - Only show this if the service has 'isExperimentalFeaturesEnabled' = true.
  - The toggle contains the metadata inbox filter.

### CODE REVIEW

File: app/views/transactions/filter.njk [hunk 1]
```diff
@@ -209,6 +209,37 @@
...
+    {% if isExperimentalFeaturesEnabled %}
+      <div class="govuk-grid-column-full">
+        <div class="govuk-grid-column-one-half">
+          <details class="govuk-details" data-module="govuk-details">
+            <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                Advanced filters
+              </span>
+            </summary>
+            <div class="govuk-details__text">
```
Comment (kbottla) [inline app/views/transactions/filter.njk:221]: Detail is currently not expanded when search is performed on metadata field. I am assuming this is expected and to be added in a separate PR ?

**Review by kbottla — APPROVED** [review]
LGTM!

File: app/views/transactions/filter.njk (hunk 1 above)
Comment (iqbalgds) [reply app/views/transactions/filter.njk:221]: Good spot - planning to do that in another PR.
Thanks.
